synapse/storage/databases/main/registration.py

```diff
@@ -2312,6 +2312,26 @@ async def add_refresh_token_to_user(
 
         return next_id
 
+    async def set_device_for_refresh_token(
+        self, user_id: str, old_device_id: str, device_id: str
+    ) -> None:
+        """Moves refresh tokens from old device to current device
+
+        Args:
+            user_id: The user of the devices.
+            old_device_id: The old device.
+            device_id: The new device ID.
+        Returns:
+            None
+        """
+
+        await self.db_pool.simple_update(
+            "refresh_tokens",
+            {"user_id": user_id, "device_id": old_device_id},
+            {"device_id": device_id},
+            "set_device_for_refresh_token",
```

Comment:
I think we usually use keyword arguments for these:

```suggestion
            keyvalues={"user_id": user_id, "device_id": old_device_id},
            updatevalues={"device_id": device_id},
            desc="set_device_for_refresh_token",
```